helm/thingsboard/Chart.yaml

```diff
@@ -18,7 +18,7 @@ apiVersion: v2
 name: thingsboard
 description: A Helm chart for Thingsboard
 type: application
-version: 0.1.3
+version: 0.1.3-rc.2
```

Comment:
rc-2?

Reply:
In our chart we have this dependency:
```
  - name: thingsboard
    version: 0.1.3-rc.1
    repository: https://midokura.github.io/thingsboard-ce-k8s
```
so I increased the number here

Reply:
Shall we rather update chart version in evp-chart repo?

This looks like a step back?

Reply:
Shall it be 0.1.4-rc.1 then?

0.1.3-rc-2 sounds like something before releasing 0.1.3, but it look like we already have 0.1.3 released.

Am I missing something?

Reply:
I think they did something weird where the chart version in the file was 0.1.3, but then they did a github release with the [0.1.3-rc.1](https://github.com/midokura/thingsboard-ce-k8s/releases/tag/thingsboard-0.1.3-rc.1) , so there wasn't actually a release for 0.1.3

Reply:
Ok, I see!

Reply:
We can release 0.1.3 then, right?

Reply:
@ricardnacher may remember, but I think we were using -rc.X because it was a custom version until thingsboard upstream update it

Reply:
I would go for 0.1.3 or 0.1.4 tbh

Reply:
As we are not changing functionality, but returning to the one we were using, I'm going to set 0.1.3